src/spec.c

```diff
@@ -1017,42 +1017,51 @@ static int parseFieldSpec(ArgsCursor *ac, IndexSpec *sp, StrongRef sp_ref, Field
   return 0;
 }
 
+FieldSpecStats FieldSpec_GetStats(const FieldSpec *fs, IndexSpec *sp){
```

Comment:
Consider a different name for this function, this is confusing.
Maybe `IndexSpec_GetFieldStats`

Reply:
Also, consider moving to field_spec_info